src/ol/imagetile.js

```diff
@@ -60,11 +60,111 @@ ol.ImageTile = function(tileCoord, state, src, crossOrigin, tileLoadFunction) {
    */
   this.tileLoadFunction_ = tileLoadFunction;
 
+  /**
+   * @private
+   * @type {number}
+   */
+  this.sequenceNumber_ = 0;
+
+
+  /**
+   * @private
+   * @type {ol.ImageTile}
+   */
+  this.outOfBandTile_ = null;
+
 };
 goog.inherits(ol.ImageTile, ol.Tile);
 
 
 /**
+ * set the url for the out-of-band tile
```

Comment:
Please use complete sentences (with capitalization and punctuation) for docs.  Same comment applies throughout.